src/test/java/seedu/address/commons/core/ReminderTest.java

```diff
@@ -0,0 +1,26 @@
+package seedu.address.commons.core;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Date;
+
+import org.junit.jupiter.api.Test;
+
+/**
+ * Contains unit test for Reminder.
+ */
+
+class ReminderTest {
+
+    @Test
+    public void getReminderMessage() {
```

Comment:
change test to `isValidReminder()` and perform validation like null checks

Reply:
The convention for test names is `methodName_testScenario_expectedResult.`